src/main/java/ResourceDiscovery/AssetObjects/AssetObject.java

```diff
@@ -126,6 +131,22 @@ public T build() {
     }
 
     // AssetObject Class Getters
+    /**
+     * Get the accountId field of this object.
```

Comment:
This is just me being a stickler, and there are many many examples of this rule not being followed, so feel free to ignore if this is too difficult or doesn't seem right, but the formal readability guidance is to use the indicative mood rather than the imperative mood under the understanding that we describe what the code does rather than tell it what to do. So "Returns the accountId..."

This is a general comment about style. For this case, I'd say that the method is actually self-descriptive enough to not require a comment at all 8-)